src/UI/Component/Button/Factory.php

```diff
@@ -209,5 +209,55 @@ public function month($default);
 	 */
 	public function tag($label, $action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     The graphical button combines the recognisability of a graphical element
+	 *     with the explicity of a text-label. It has an obtrusive presentation that
+	 *     indicate important actions on the screen and make them hard to overlook.
+	 *
+	 *   composition: >
+	 *     The graphical button consists of an icon or glyph and a (very short) text.
+	 *
+	 *   effect: >
+	 *     Both, icon/glyph and text, form an entity an can only be operated together.
```

Comment:
pls delete "Both, icon/glyph and text, form an entity an can only be operated together."

A button is always operated as a whole.

Reply:
Removed that.